index.yaml

```diff
@@ -1,19 +1,49 @@
 apiVersion: v1
```

Comment:
Just a reminder to revert changes to the `index.yaml` and any chart tarballs before committing. :reminder_ribbon: